lib/functions/oauth_providers/gitlab.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * TestLink Open Source Project - http://testlink.sourceforge.net/ 
+ * This script is distributed under the GNU General Public License 2 or later. 
+ *
+ * @filesource  github.php
```

Comment:
Commenti is wrong the file name is not github

Reply:
Thx,it was fixed!